config/tests/jobs/jobs_test.go

```diff
@@ -827,7 +827,9 @@ func checkScenarioArgs(jobName, imageName string, args []string) error {
 	}
 
 	expectedExtract := 1
-	if sharedBuilds || nodeE2e || builds {
+	if sharedBuilds || nodeE2e {
+		expectedExtract = 0
+	} else if builds && !extracts {
```

Comment:
this is from https://github.com/kubernetes/test-infra/blob/master/jobs/config_test.py#L323-L324
/shrug